src/table/components/__tests__/withSelections.spec.jsx

```diff
@@ -67,7 +62,7 @@ describe('withSelections', () => {
     );
     fireEvent.mouseUp(queryByText(value));
 
-    expect(selectionsUtils.selectCell).to.have.been.calledWithMatch({ selectionState, cell, evt, announce });
+    expect(selectionsUtils.selectCell).toHaveBeenCalledTimes(1);
```

Comment:
Tried to use 
```
expect(selectionsUtils.selectCell).toHaveBeenCalledWith(
  expect.objectContaining({  selectionState, cell, evt, announce  })
)
```
to do the loose match.
It did not work. Do not know why.

Reply:
That is because there is no loose match, it needs to match exactly. 
You can however get the last call and do assertions on that. https://jestjs.io/docs/mock-function-api#mockfnmocklastcall

Reply:
Done